core/src/main/resources/openapi/eth2signer.yaml

```diff
@@ -0,0 +1,92 @@
+openapi: 3.0.0
+info:
+  title: 'Eth2Signer'
+  description: 'Sign Eth2 Artifacts'
+  version: '@VERSION@'
+  license:
+    name: 'Apache 2.0'
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+
+servers:
+  - url: /
+  - url: http://localhost:9000/
+
+paths:
+  /signer/sign/{publicKey}:
+    post:
+      tags:
+      - 'Signing'
+      summary: 'Signs data for public key'
+      description: 'Signs data for the public key specified as part of the URL and returns the signature'
+      operationId: 'signForPublicKey'
+      parameters:
+      - name: 'publicKey'
+        in: 'path'
+        required: true
+        description: 'Public Key for which data to sign'
+        type: 'string'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                signingRoot:
+                  type: string
+              example:
+                signingRoot: '0x48656c6c6f2c20776f726c6421'
+      responses:
+        '200':
+          description: 'hex encoded string of signature'
+          content:
+            text/plain; charset=utf-8:
+              schema:
+                type: string
+                example: '0xb3baa751d0a9132cfe93e4e3d5ff9075111100e3789dca219ade5a24d27e19d16b3353149da1833e9b691bb38634e8dc04469be7032132906c927d7e1a49b414730612877bc6b2810c8f202daf793d1ab0d6b5cb21d52f9e52e883859887a5d9'
+        '404':
+          description: 'Public Key not found'
+        '400':
+          description: 'Bad request format'
+        '500':
+          description: 'Internal Eth2Signer server error'
+
+  /signer/publicKeys:
+    get:
+      tags:
+        - 'Public Key'
+      summary: 'List of available Public Keys'
+      description: 'Returns the public keys for the private keys that have been loaded into Eth2Signer'
+      operationId: 'getPublicKeys'
+      responses:
+        '200':
```

Comment:
When use the try it now, I'm getting message in the response body area "Unrecognized response type; displaying content as text."

Reply:
Yes, I noticed that as well, I will debug it with curl to figure out whats vertx sending (we do set json type in server).